tests/functional/api/view_route_by_content_test.py

```diff
@@ -0,0 +1,26 @@
+from urllib.parse import quote_plus
+
+import pytest
+
+from tests.conftest import assert_cache_control
+
+
+class TestRouteByContent:
+    def test_proxy_html(self, test_app, html_response):
```

Comment:
```suggestion
    @pytest.mark.usefixtures("html_response")
    def test_proxy_html(self, test_app):
```

We normally use mark when we have fixtures that setup a situation rather than return a value. In fact, we intentionally _don't_ return a value in fixtures where we are trying to setup a situation, to help you tell the fact by looking at the fixture definition (no return value = pure side effect fixture)

This helps in a few ways:

 * It's a bit baffling if you look at the code and think "This arg isn't used, I'll remove it" then the test breaks
 * The value is just None, so if you debug, it's not clear why this happened